aspnet/fundamentals/logging.rst

```diff
@@ -1,8 +1,202 @@
-.. include:: /../common/stub-topic.txt
+Logging
+=======
 
-|stub-icon| Logging
-===================
+By `Steve Smith`_
 
-.. include:: /../common/stub-notice.txt
+ASP.NET 5 has built-in support for logging, and allows developers to easily leverage their preferred logging framework's functionality as well. Implementing logging in your application requires a minimal amount of setup code. Once this is in place, logging can be added wherever it is desired.
+
+In this article:
+	- `Implementing Logging in your Application`_
+	- `Configuring Logging in your Application`_
+	
+`View or download sample from GitHub <https://github.com/aspnet/Home/tree/dev/samples>`_.
+
+Implementing Logging in your Application
+----------------------------------------
+
+Adding logging to your application is done by requesting either an ``ILoggerFactory`` or an ``ILogger<T>`` via :doc:`dependency-injection`. If an ``ILoggerFactory`` is requested, a logger must be created using its ``CreateLogger`` method. The following example shows how to do this within the ``Configure`` method in the ``Startup`` class:
+
```

Comment:
I'd probably introduce this like "Adding logging **to a component** in your application...". You still need to do the configuration tasks (or use the template).